.eslintrc

```diff
@@ -61,10 +61,11 @@
       "tabWidth": 2,
       "ignoreTrailingComments": true,
       "ignoreUrls": true,
+      "ignoreStrings": true,
```

Comment:
are these changes still required?

Reply:
I think this change makes a lot of sense.

Reply:
@danseethaler FYI - he was re-reviewing the linting rules

Reply:
#4968 Removes the `max-len` rule in favor of the `eslint-plugin-prettier` which will enforce the max-length when it makes sense.